neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -65,7 +66,7 @@ def home(self, request):
             if "method" not in body:
                 raise JsonRpcError("Field 'method' is missing", JsonRpcErrors.INVALID_REQUEST)
 
-            params = body["params"] if "params" in body else None
+            params = ast.literal_eval(body["params"]) if "params" in body else None
```

Comment:
wut, whats going on here? json does correct parsing of the string already, what's the use-case for this?